rustworkx-core/src/generators/hexagonal_lattice_graph.rs

```diff
@@ -78,16 +78,24 @@ pub fn hexagonal_lattice_graph<G, T, F, H, M>(
     mut default_node_weight: F,
     mut default_edge_weight: H,
     bidirectional: bool,
+    periodic: bool,
+    with_positions: Option<F>,
```

Comment:
This is a breaking API change on the Rust side. On the Python side, you can use optional arguments but on Rust there is not much we can do.